README.md

```diff
@@ -1,2 +1,206 @@
+  
+ The build is minified and the filenames include the hashes.\ 
+ Your app is ready to be deployed! 
+  
+ See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information. 
+  
+ ### `npm run eject` 
+  
+ **Note: this is a one-way operation. Once you `eject`, you can't go back!** 
+  
+ If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project. 
+  
+ Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own. 
+  
+ You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it. 
+  
+ ## Learn More 
+  
+ You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started). 
+  
+ To learn React, check out the [React documentation](https://reactjs.org/). 
+  
+ ### Usage 
+  
+ To run the project, execute the following command: 
+  
+ - `npm start` 
+  
+ ### Run tests 
+  
+ To run tests, run the following command: 
+  
+ - `npm test` 
+  
+ <!-- AUTHORS --> 
+  
+ ## 👥 Author <a name="authors"></a> 
+  
+ 👤 **Nnamani Stanley Wisdom aka StarMindz** 
+  
+ - GitHub: [@StarMindz](https://github.com/Starmindz) 
+ - LinkedIn: [Stanley Nnamani](https://www.linkedin.com/in/stanley-nnamani) 
+  
+ <p align="right">(<a href="#readme-top">back to top</a>)</p> 
+  
+ <!-- FUTURE FEATURES --> 
+-I will use more of NASA's API to implement more feature in this WebApp
+-I will be adding a Mars Rover Explore page soon
+
+  
+ <!-- CONTRIBUTING --> 
+  
+ ## 🤝 Contributing <a name="contributing"></a> 
+  
+ Contributions, issues, and feature requests are welcome! 
+  
+ Feel free to check the [issues page](../../issues/). 
+  
+ <p align="right">(<a href="#readme-top">back to top</a>)</p> 
+  
+ <!-- SUPPORT --> 
+  
+ ## ⭐️ Show your support <a name="support"></a> 
+  
+ If you like this project please give it a star! 
+  
+ <p align="right">(<a href="#readme-top">back to top</a>)</p> 
+  
+ <!-- ACKNOWLEDGEMENTS --> 
+  
+ ## 🙏 Acknowledgments <a name="acknowledgements"></a> 
+  
+ - I drew some design inspiration from [Nelson Sakwa](https://www.behance.net/gallery/31579789/Ballhead-App-%28Free-PSDs%29) 
+  
+ <p align="right">(<a href="#readme-top">back to top</a>)</p> 
+  
+ <!-- LICENSE --> 
+  
+ ## 📝 License <a name="license"></a> 
+  
+ This project is [MIT](./LICENSE) licensed. 
+  
+ <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] [REQUIRED] Also, as I have recommended above, kindly ensure you have properly embedded the links so that your project looks more professional.
### Your README
<img width="794" alt="Screenshot 2022-12-03 at 23 44 25" src="https://user-images.githubusercontent.com/58692590/205465642-94c764da-5d56-4f2d-b64f-0dbd833e1e29.png">

### Expected template
<img width="521" alt="Screenshot 2022-12-03 at 23 59 40" src="https://user-images.githubusercontent.com/58692590/205465679-a64aac05-1a7d-4173-9d14-5cc1dca7ceeb.png">